iguana.corecontroller/src/test/java/org/aksw/iguana/cc/utils/FileUtilsTest.java

```diff
@@ -29,6 +51,22 @@ public void readLineAtTest() throws IOException {
         assertEquals("dfe", FileUtils.readLineAt(4, f));
         //read line at -1
         assertEquals("", FileUtils.readLineAt(-1, f));
+
+        File f1 = new File("src/test/resources/readLineTestFile1.txt");
+        File f2 = new File("src/test/resources/readLineTestFile2.txt");
```

Comment:
please separate the processing of the files. Additionally, define the mappings sth like this: 
```
Map<String, Integer> left = ImmutableMap.of(
0, "line 1",
1, "line 2", 
... )
```

I would suggest to define a lambda function that get's such a map and a file path as input and preforms all the checks. Then you can call the lambda for the 4 files tested in this test case. 

Alternatively it would also make sense to break them down into sub-test cases. Please investigate what the possiblities are here. I am not into junit. 

In the report it should roughly look like this:
```
✅ read line at
  ✅ fileUtils.txt
  ✅ readLineTestFile1.txt
  ✅ readLineTestFile2.txt
  ✅ readLineTestFile3.txt
```

Reply:
will be fixed in a separate pr